src/runtime/eventbinding.cs

```diff
@@ -118,7 +119,20 @@ public new static void tp_dealloc(IntPtr ob)
         {
             var self = (EventBinding)GetManagedObject(ob);
             Runtime.XDecref(self.target);
-            ExtensionType.FinalizeObject(self);
+            self.Dealloc();
+        }
+
+        public static int tp_clear(IntPtr ob)
+        {
+            var self = (EventBinding)GetManagedObject(ob);
+            Runtime.Py_CLEAR(ref self.target);
+            return 0;
+        }
+
+        protected override void OnSave()
```

Comment:
No, because `target` will decrease a reference in `tp_clear` when `TypeManager.RemoveTypes` called, it needs to be increase at once make sure it lived when it `StashPop` one time, you can think it's like its constructor to Incref it.